src/headless/converse-chatboxes.js

```diff
@@ -1258,6 +1287,9 @@ converse.plugins.add('converse-chatboxes', {
                             !chatbox.handleChatMarker(stanza, from_jid, is_carbon, is_roster_contact, is_mam)) {
 
                         const attrs = await chatbox.getMessageAttributesFromStanza(stanza, original_stanza);
+                        if (!utils.isHeadlineMessage(_converse, stanza)) {
```

Comment:
I think the `isHeadlineMessage` check should go into `set_editable`, that way you don't have to always remember doing this check before calling it.